tests/trainer/test_trainer_tricks.py

```diff
@@ -1,174 +0,0 @@
-# Copyright The PyTorch Lightning team.
```

Comment:
ok, I see how it came, but pls lets improve these tests :]

Reply:
done!